### PR TITLE
Sscs 9871

### PR DESCRIPTION
**Pull request by gmikerlemos**
Please remove this line and everything above and fill the following sections:

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SSCS-9871

### Change description ###

Child Support - Update Reissue Further Evidence Event

**Does this PR introduce a breaking change?** (check one with "x")

```
[x ] Yes
[ ] No
```

### CODE REVIEW

File: benefit/data/sheets/CaseField/CaseField.json [hunk 1]
```diff
@@ -407,5 +407,7 @@
...
-  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "otherPartyHearingRecordingReqUi", "Label": "Other party hearing recording requests", "FieldType": "Collection", "FieldTypeParameter": "otherPartyHearingRecordingReqUi", "SecurityClassification": "PUBLIC"}
+  {"LiveFrom": "01/11/2021", "CaseTypeID": "Benefit", "ID": "otherPartyHearingRecordingReqUi", "Label": "Other party hearing recording requests", "FieldType": "Collection", "FieldTypeParameter": "otherPartyHearingRecordingReqUi", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/12/2021", "CaseTypeID": "Benefit", "ID": "otherPartyOptions", "Label": "Resend to other party", "FieldType": "Collection", "FieldTypeParameter": "OtherPartyOption", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "01/12/2021", "CaseTypeID": "Benefit", "ID": "showReissueToOtherPartyUiSection", "Label": "Show Reissue To Other Party Ui Section", "FieldType": "YesOrNo", "SecurityClassification": "PUBLIC"}
```
Comment (ryandac) [inline benefit/data/sheets/CaseField/CaseField.json:412]: Label text should have initial cap to be consistent with other labels.

Comment (gmikerlemos) [reply benefit/data/sheets/CaseField/CaseField.json:412]: I'll fix that but that field will never be displayed.

File: benefit/data/sheets/ComplexTypes/ComplexTypes.json [hunk 2]
```diff
@@ -437,6 +437,8 @@
...
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReq", "ListElementCode": "otherPartyId", "FieldType": "Text", "ElementLabel": "Hearing ID", "SecurityClassification": "PUBLIC"},
   {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "otherPartyName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
-  {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "hearingRecordingStatus", "FieldType": "DynamicList", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"}
-
+  {"LiveFrom": "18/06/2019", "ID": "otherPartyHearingRecordingReqUi", "ListElementCode": "hearingRecordingStatus", "FieldType": "DynamicList", "ElementLabel": "Status", "SecurityClassification": "PUBLIC"},
+  {"LiveFrom": "18/12/2021", "ID": "OtherPartyOption", "ListElementCode": "otherPartyOptionName", "FieldType": "Text", "ElementLabel": "Other party name", "SecurityClassification": "PUBLIC"},
```
Comment (jackmaloney) [inline benefit/data/sheets/ComplexTypes/ComplexTypes.json:441]: Maybe it doesn't matter but normally the ID's start with lower case letter

**Review by jackmaloney — APPROVED** [review]
Approved with very minor comment

File: benefit/data/sheets/ComplexTypes/ComplexTypes.json (hunk 2 above)
Comment (gmikerlemos) [reply benefit/data/sheets/ComplexTypes/ComplexTypes.json:441]: it doesn't, as I tested and it worked. I followed the SscsDocument complex type example

**Pull request merged**